plugin-sources/orchardist/scripts/session-index.py

```
@@ -0,0 +1,270 @@
+        on_disk.add(session_id)
+        mtime = os.path.getmtime(f)
+
+        if session_id in existing and existing[session_id] == mtime:
+            skipped += 1
+            continue
+
+        project = os.path.basename(os.path.dirname(f))
+        try:
+            messages = extract_user_messages(f)
+        except Exception:
+            continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)